lib/config/presets/local/index.ts

```diff
@@ -3,6 +3,7 @@ import { GlobalConfig } from '../../global';
 import * as azure from '../azure';
 import * as bitbucket from '../bitbucket';
 import * as bitbucketServer from '../bitbucket-server';
+import * as gerrit from '../gerrit';
```

Comment:
```suggestion
import * as local from './common';
```

Reply:
- #19025

Reply:
Okay, i've rebased to current main to get these changes and removed my own gerrit-"local" preset implementation.

But there is now a problem: 

the existing `fetchJSONFile` from presets/local/common.ts don't pass thru the `tag` down to the 

`raw = await platform.getRawFile(fileName, repo);`

call.

Reply:
- #19102